src/components/Results/Results.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { CardsGrid, Card, Thumbnail, ThumbnailImg, InfoArea } from './Results.components';
+
+const Results = ({ searchResultItems }) => {
```

Comment:
I would rename this variable to something more descriptive of what it is, for example, `videoCards` because results is very generic